scripts/scripts.js

```diff
@@ -31,13 +31,28 @@ function buildHeroBlock(main) {
   }
 }
 
+/**
+ * Builds breadcrumb block and prepends to main in a new section.
+ * @param {Element} main The container element
+ */
+function buildBreadcrumbBlock(main) {
+  const title = document.querySelector('head title');
```

Comment:
I imagine we will eventually want to refine the criteria for when breadcrumb is/is not added, probably based on a metadata property, but we can do so later

Reply:
actually, let's do that now. Here's what I think the criteria should be:

* if there is a metadata property hideBreadcrumb!=='false` then we skip it 
* if it's the home page (path === '/') then we skip it